### PR TITLE
DDINGI-1306: Revised Data Paging/Totals

### PR DESCRIPTION
**Pull request by Ramky-Infoblox**
DDINGI-1306: Revised Data Paging/Totals

Enhancement to the existing pagination with resultset count with enable/disable count feature.

### CODE REVIEW

File: query/collection_operators.proto [hunk 1]
```diff
@@ -202,6 +202,9 @@ message Pagination {
     // The service may impose maximum value.
     // If omitted the service may impose a default value.
     int32 limit = 3;
+    //The bool value enables/disables the record count.
+    //The default setting disables the record count.
+    bool is_total_size = 4;
```
Comment (chinmayb) [inline query/collection_operators.proto:207]: How about this ? cc @pnagaraj 
```suggestion
    // The bool value enables/disables the record count.
    // The default setting disables the record count.
    bool is_total_count_required = 4;
```

File: query/collection_operators.proto [hunk 2]
```diff
@@ -220,6 +223,8 @@ message PageInfo {
     // The service may optionally include the offset of the next page of resources.
     // A null value indicates no more pages.
     int32 offset = 3;
+    //Resultset record count.
+    int64 total_size = 4;
```
Comment (chinmayb) [inline query/collection_operators.proto:227]:
```suggestion
    // total_size indicates the total records present.
    int64 total_size = 4;
```

File: query/collection_operators.proto (hunk 1 above)
Comment (Ramky-Infoblox) [reply query/collection_operators.proto:207]: @chinmayb - This is not a required field. 
As comment says it is just to enable or disable the record count.

File: query/collection_operators.proto (hunk 2 above)
Comment (Ramky-Infoblox) [reply query/collection_operators.proto:227]: sure will modify

File: query/collection_operators.proto (hunk 1 above)
Comment (Ramky-Infoblox) [reply query/collection_operators.proto:207]: for comment space will modify it

Comment (chinmayb) [reply query/collection_operators.proto:207]: the user asks this, hence I was asking that way. It doesn't indicate that its a required field or anything. More like
```
bool is_total_size_needed
```

Comment (Ramky-Infoblox) [reply query/collection_operators.proto:207]: Sure

**Review by chinmayb — APPROVED** [review]

**Pull request merged**